introduction.md

```diff
@@ -0,0 +1 @@
+Hi, I'm user jeffesterby.  I'm new to the dojo_rules repo and I'm excited to be here I suppose.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.